browser/admin/src/AdminSocketAnalytics.js

```diff
@@ -524,6 +586,26 @@ var AdminSocketAnalytics = AdminSocketBase.extend({
 				this._updateNetGraph();
 			}
 		}
+		else if (textMsg.startsWith('connection_activity')) {
+			textMsg = textMsg.split(' ')[1];
+			if (textMsg.endsWith(',')) {
```

Comment:
Quite a chunk of copy/paste here - lets split this off for the three chunks here into a helper function.

Reply:
Resolved w/ PR #10574